crates/astria-composer/tests/blackbox/helper/mod.rs

```diff
@@ -50,11 +51,6 @@ pub struct TestComposer {
 pub async fn spawn_composer(rollup_ids: &[&str]) -> TestComposer {
     Lazy::force(&TELEMETRY);
 
-    assert!(
-        !rollup_ids.is_empty(),
-        "must provide at least one rollup ID for tests"
-    );
-
```

Comment:
Removed this because we can have deployments where there are no geth collectors but only one gRPC collector.